api/models/task.py

```diff
@@ -420,6 +423,10 @@ class Task(Base):
     has_predictions_upload = db.Column(db.Boolean, default=False)
     predictions_upload_instructions_md = db.Column(db.Text)
 
+    default_metric_weights = db.orm.relationship(
+        "DefaultMetricWeight", backref="task", lazy="dynamic"
+    )
```

Comment:
just curious - is there a reason why we don't use orm.relationships to allow parents to access their children in a one-to-many/many-to-many relationship? 

eg.
https://github.com/facebookresearch/dynabench/blob/79981d543189d8b9a28adb8f17c0e2b07c5767cf/api/controllers/tasks.py#L188-L190

in contrast to the examples in sqlalchemy's relationship documentation (which I temporarily implemented here as an example, and allows one to access the `default_metric_weights` of a specific task using `some_task.default_metric_weights`

tried to look up performance differences but couldn't find anything, so thought I'd ask

Reply:
I'm not aware of a good reason for this. We could ask Douwe when he gets back.